### PR TITLE
docs: update docs to v1.10.7 CLI commands

### PR DESCRIPTION
**Pull request by Zachinquarantine**

### CODE REVIEW

**Review by karalabe — CHANGES_REQUESTED** [review]
You need to get rid of all the spurious whitespaces. Perhaps check your editor, seems you have some auto-indentation in place that makes things go weird.

File: docs/_interface/Command-Line-Options.md [hunk 1]
```diff
@@ -226,5 +226,5 @@ MISC OPTIONS:
 
 
 COPYRIGHT:
-   Copyright 2013-2021 The go-ethereum Authors
```
Comment (MariusVanDerWijden) [inline docs/_interface/Command-Line-Options.md:229]: Please don't delete this tab

**Review by MariusVanDerWijden — APPROVED** [review]
LGTM, just a tiny nitpick

**Comment by Zachinquarantine** [comment]
closing this because v1.10.8 is coming out very soon and I'll send a PR for the docs then instead...

**Pull request closed without merging**